packages/block-library/src/button/edit.native.js

```diff
@@ -189,6 +189,7 @@ class ButtonEdit extends Component {
 		}
 		return (
 			getColorAndStyleProps( attributes ).style?.backgroundColor ||
```

Comment:
I see that the `getColorAndStyleProps` does something under the hood and it is not the simplest function. WDYT about calling it once with the assignment to the const?

```js
  const colorAndStyleProps = getColorAndStyleProps( attributes );

  return (
    colorAndStyleProps.style?. backgroundColor ||
    colorAndStyleProps.style?.background ||
    ...
```

Reply:
Makes sense!